app/konnect/runtime-manager/kic.md

```diff
@@ -43,6 +43,9 @@ If you don't have an existing KIC deployment, you need the following before usin
 * `kubectl` or `oc` (if you're working with OpenShift) installed and configured to communicate with your Kubernetes TLS
 * [Helm 3](https://helm.sh/docs/intro/install/) installed
 
+Also, as {{site.kic_product_name}} calls the {{site.konnect_short_name}}'s APIs, outbound traffic from the
+{{site.kic_product_name}}'s pods must be allowed to reach {{site.konnect_short_name}}'s `*.konghq.com` hosts.
```

Comment:
```suggestion
* Because {{site.kic_product_name}} calls {{site.konnect_short_name}}'s APIs, outbound traffic from {{site.kic_product_name}}'s pods must be allowed to reach {{site.konnect_short_name}}'s `*.konghq.com` hosts.
```

Reply:
Should we also add this information to https://deploy-preview-5781--kongdocs.netlify.app/konnect/network/ under "Hostnames"?

Reply:
Yes, I think that's a good point. Updated the `/konnect/network` accordingly and linked to it from the 2 articles edited.